### ISSUE TITLE
Ship with ID has a mass of 0.0, not creating a ShipObject

### ISSUE THREAD

**Issue by KrokusAstra** [ValkyrienSkies/Create-Interactive-Issues#108]
At some point my world became laggy, due to spam in log file alot of this trash
I can't normally play, everything is lagging, i can't even eat the FOOD. 1 hour ago i'm easily travelled across the world, killed mobs and other things without single lag. And now everything is went downhill, i barely can play. Opening a door or sleeping sometimes costs me 2-5 minutes until single-player game actually un-lag and allow me to do that.
![image](https://github.com/user-attachments/assets/46098718-279f-4fd3-ae79-310d1da81194)

My latest log file is whopping 1 million kb. That is insane!
![image](https://github.com/user-attachments/assets/58ce2aba-f187-4581-8a3c-434b6a791792)

**Comment by tivvo** [comment]
bump, my servers gettin borked'
from what i read though in https://github.com/ValkyrienSkies/Create-Interactive-Issues/issues/85, it's a valk skies issue by the looks of it and i hope they end up fixing it or interactive fixes it because my god

**Comment by Twor** [comment]
### I am also encountering the same issue.

![屏幕截图 2024-10-04 165424](https://github.com/user-attachments/assets/003c80b4-5c2e-43fd-ae0a-405fd1bf7774)

**Comment by OstiumAxioma** [comment]
Seems like not one is trying to fix this issue? I believe it's a bug relates to Create:Interative

**Comment by tivvo** [comment]
as @OstiumAxioma said, and through my own digging it's a issue with create: interactive, i did find a temporary fix for it and that is to install VMod (https://modrinth.com/mod/vmod), and kill @v, which is all valkyriean skies entities

only issue is; well, all your vs2 entities will die, i recommend **for now, to uninstall create interactive as its in massive beta, because it also breaks minecart contraption chests, which is a whole other issue thats more impactful then creating 217mb logs**

tl;dr, to fix the issue install vmod and kill @v or uninstall create: interactive

**Comment by Knewest** [comment]
I am curious to how often `@v` gets tagged in messages-- I am also encountering this issue, by the way.

**Comment by LordRomzessV** [comment]
Um... I removed create:interactive from my server, but ShipObject entities are still attacking server resources and the log file. And I'm playing on version 1.19.2, which has no vmod. Is there any other way to remove these "weightless ships"?

**Comment by xiewuzhiying** [comment]
Fixed in development version

**Comment by blockninja124** [comment]
You can get the latest dev version of create interactive from the Valkyrien Skies discord here: https://discord.com/channels/244934352092397568/1327007036998680688
(Look for the #interactive-nightlies channel)

Unfortunately the dev versions are only for 1.20 (forge or fabric) right now, but hopefully we can get versions for 1.18 and 1.19 soon ish. 

These versions will eventually be released on modrinth (and hopefully Curseforge, after we remove the obfuscation). But we aren't _quite_ ready to release them yet. (Probably after we get 1.18 and 1.19 working and fix some more bugs we will have a public release). 

**Currently the dev version fixes:**
- Config not working (you can now configure contraptions to not need the sticker)
- Interact-me sticker having no texture
- Un-interactive contraptions have no inventory
- Massless ships

**Comment by Plasman888** [comment]
Oh can i check, i also have the mod removed but still encounter this error, if i install the beta then remove it would it permenantly remove the error or would it still be there?

**Comment by blockninja124** [comment]
The dev version won't delete massless ships, it will just stop creating them. If you're on 1.18 or 1.20 you can install VMod, which _will_ automatically delete massless ships.

**Comment by arlexTech** [comment]
My server has a few weeks old and logs weight 180 GB of theses, even if i disbled the mod a week ago it's still spamming theses in the console

**Comment by blockninja124** [comment]
Interactive release 1.1.1 stopped creating new massless ships, but you will need to install VMod to delete previously created massless ships.
